torch/export/_unlift.py

```diff
@@ -153,6 +190,7 @@ def _unlift(
     state_dict: Dict[str, Any],
     constants: Dict[str, Any],
     forward_arg_names: Optional[List[str]] = None,
+    mutated_inputs: Optional[Dict[torch.fx.Node, torch.fx.Node]] = None,
```

Comment:
The style of this parameter kind of conflicts with `mutated_outputs` as one is `List[Optionl[str]]` and the other is `Optional[Dict[torch.fx.Node, torch.fx.Node]]`. It is because we need to pass the information of `graph signature` to this function. We need to get the mapping between `output nodes` and `mutated input nodes` for aten operations with `out` parameter.

Reply:
cc @angelayi + @tugsbayasgalan, would be good to get your input.

The "new" thing happening in this PR is that Eikan wants to support mutations on normal forward inputs - I think this unlifting logic only knows to apply mutations to params/buffers, so we'll probably need to update it to also support mutations on inputs that are neither params nor buffers.

Reply:
We should already support unlifting input mutations here: https://github.com/pytorch/pytorch/blob/main/torch/export/_unlift.py#L302-L310

Reply:
@angelayi , I will check how to leverage the code that you shared.

Reply:
Couldn't another solution be to fix https://github.com/pytorch/pytorch/blob/main/torch/export/_unlift.py#L102-L111 so that we return the output of the copy_ ?

Reply:
We are on the same page now. I have fixed it in this PR and will refine the code to align the solution.